packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.tsx

```diff
@@ -159,7 +140,7 @@ export class BaseFloatingPicker<T, P extends IBaseFloatingPickerProps<T>> extend
     return (
       <div
         ref={ this.root }
-        className={ css('ms-BasePicker', className ? className : '') }
+        className={ css('ms-BaseFloatingPicker', className ? className : '') }
```

Comment:
Technically this would be a breaking change, you should probably keep the old class name in case anyone relies on it still.

Reply:
I'll add the new and keep the old